client/ayon_equalizer/plugins/publish/extract_matchmove_script_maya.py

```diff
@@ -103,36 +105,75 @@ def process(self, instance: pyblish.api.Instance):
-                        for model in model_list:
-                            tde4.set3DModelSelectionFlag(point_group, model, 0)
-
-            file_path = Path(staging_dir) / "maya_export.mel"
-            status = exporter._maya_export_mel_file(
-                file_path.as_posix(),  # staging path
-                point_group,  # camera point group
-                [c["id"] for c in instance.data["cameras"] if c["enabled"]],
-                model_selection,  # model selection mode
-                overscan_width,
-                overscan_height,
-                1 if attr_data["export_uv_textures"] else 0,
-                scale_factor,
-                offset,  # start frame
-                1 if attr_data["hide_reference_frame"] else 0)
 
-        if status != 1:
-            raise KnownPublishError("Export failed.")
+                    # clear all other model selections
+                    for model in model_list:
+                        tde4.set3DModelSelectionFlag(point_group, model, 0)
+
+            file_path = Path(staging_dir) / "maya_export"
+            if instance.context.data.get("tde4_version"):
+                self.log.debug(f"Exporting to: {file_path.as_posix()}")
+
+            # create representation data
+            if "representations" not in instance.data:
+                instance.data["representations"] = []
+
+            if instance.context.data["tde4_version"].major == EQUALIZER_7:
+                status = exporter._maya_export_mel_file(  # noqa: SLF001
+                    f"{file_path.as_posix()}.mel",
+                    point_group,
+                    [
+                        c["id"] for c in instance.data["cameras"]
+                        if c["enabled"]
+                    ],
+                    model_selection,
+                    overscan_width,
+                    overscan_height,
+                    1 if attr_data["export_uv_textures"] else 0,
+                    scale_factor,
+                    offset,
+                    1 if attr_data["hide_reference_frame"] else 0,
+                )
+
+                representation = {
+                    "name": "matchmoveScript",
+                    "ext": "mel",
+                    "files": file_path.name,
+                    "stagingDir": staging_dir,
+                }
+            elif instance.context.data["tde4_version"].major == EQUALIZER_8:
+                exporter.script_version = "4.7"
+                status, npoly_warning = exporter._maya_export_python_file(  # noqa: SLF001
+                    file_path.as_posix(),  # staging path,
+                    point_group,  # camera point group,
+                    [
+                        c["id"] for c in instance.data["cameras"]
+                        if c["enabled"]
+                    ],
+                    model_selection,
+                    overscan_width,
+                    overscan_height,
+                    1 if attr_data["export_uv_textures"] else 0,
```

Comment:
All these "1 or 0" can be simplified converting to bool and int:
```suggestion
                    int(bool(attr_data["export_uv_textures"])),
```

Reply:
maybe little simplified but at least for me it is more readable and explicit that it really requires `1` an `0` and not bool